app.js

```diff
@@ -0,0 +1,170 @@
+const express = require('express');
+// const { where } = require('sequelize/types');
+const { Customers, Order, Item, Sequelize } = require('./models');
+
+
+const app = express();
+const PORT = 4000;
+
+
+app.use(express.json());
+
+
+app.get('/customers', async (req, res) => {
+    const result = await Customers.findAll().catch((err) => {
+        res.status(500).send(`something wnt wrong ${err}`);
+    })
+
+    res.status(200).send(result);
+})
+
+app.post('/customers', async (req, res) => {
+    const { firstName, lastName, email, password, address } = req.body;
+    const result = await Customers.create({
+        firstName: firstName,
+        lastName: lastName,
+        email: email,
+        password: password,
```

Comment:
We can call a hash function instead of storing strings for password.
https://www.npmjs.com/package/bcrypt
```suggestion
        password: hash(password),
```